config/clients/java/template/Java/libraries/native/build.gradle.mustache

```diff
@@ -0,0 +1,124 @@
+apply plugin: 'idea'
+apply plugin: 'eclipse'
+apply plugin: 'com.diffplug.spotless'
+
+group = '{{groupId}}'
+version = '{{artifactVersion}}'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.diffplug.spotless:spotless-plugin-gradle:6.11.0'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply plugin: 'java'
+apply plugin: 'maven-publish'
```

Comment:
```suggestion
plugins {
    id 'java'
    id 'maven-publish'
    id 'idea'
    id 'eclipse'
    id 'com.diffplug.spotless' version '6.+'
}

group = '{{groupId}}'
version = '{{artifactVersion}}'

repositories {
    mavenCentral()
}
```

Reply:
Change from "legacy" plugin application to modern style

e.g. https://plugins.gradle.org/plugin/com.diffplug.spotless